pyproject.toml

```
@@ -9,18 +9,58 @@ keywords = ['conda', 'security', 'vulnerabilitiy scanner']
 repository = "https://github.com/exasol/conda-vulnerability-scanner"
 homepage = "https://exasol.github.io/conda-vulnerability-scanner/main/"
 
-
 [tool.poetry.dependencies]
 python = ">=3.8,<4.0"
 
 [tool.poetry.dev-dependencies]
 pytest = "^7.2.0"
 nox = "^2022.8.7"
-exasol-sphinx-github-pages-generator = { git = "https://github.com/exasol/sphinx-github-pages-generator.git", rev = "16143e6acac138875730eb4a478cbca663bf6b48"}
 gitpython = ">=2.1.0"
 toml = ">=0.10.2"
-pre-commit = "^2.20.0"
+pre-commit = ">=2.20.0"
+exasol-toolbox = ">=0.15.0"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+# python-toolbox config:
+
+[tool.coverage.run]
+relative_files = true
+source = [
+    "exasol",
+]
```

here the module for whom coverage should be accounted for should be placed.
```suggestion
source = [
    "conda_vulnerability_scanner",
]
```